.gitignore

```diff
@@ -137,3 +137,10 @@ Gemfile.local
 # Typings
 flow-typed
 config/cable.yml
+
+# Yarn
+.yarn/cache
+.yarn/unplugged
+.yarn/build-state.yml
+.pnp.*
+package-lock.json
```

Comment:
If we're not supposed to have a package-lock, we should not ignore it. If somebody ever commits it, it's a sign they did something wrong.

Reply:
We should, like any other file that shouldn't be commited. There are instructions to follow and if there's a change in package.json and not in yarn.lock, we will address in the PR review